core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -188,6 +180,17 @@ oppia.directive('audioBar', [
             }
           };
 
+          $scope.track = {             
+            progress: function(progress) {
+              if (arguments.length){
+                AudioPlayerService.setProgress(progress * 1.0 / 100);
```

Comment:
Is the `* 1.0` necessary ?

Reply:
No, it isn't that was a check which I forgot to remove :sweat_smile: